### PR TITLE
[SDKTECHNO-86] import pandas fails in jupyter

### PR DESCRIPTION
**Pull request by finalspy**

### CODE REVIEW

**Comment by EtienneSIG** [comment]
Pandas library works fine. I made several test on it. 
image

**Review by EtienneSIG — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -10,4 +10,6 @@ tmp-zip
 # Ignore Node.js technologies build and cache directories
 node_modules/
 .cache/
-.bundle/
\ No newline at end of file
+.bundle/
+jupyter-pandas.zip
```
Comment (PierreLeresteux) [inline .gitignore:14]: Why ? it's just some local files it's it ?

Comment (finalspy) [reply .gitignore:14]: you're right

Comment (PierreLeresteux) [reply .gitignore]: ALWAYS !

**Review by NicolasCD — COMMENTED** [review]
OK for me

**Review by NicolasCD — APPROVED** [review]

**Pull request merged**